pkg/storage/couchdb/couchdbstore_test.go

```diff
@@ -27,7 +27,12 @@ const (
 
 // For these unit tests to run, you must ensure you have a CouchDB instance running at the URL specified in couchDBURL.
 // 'make unit-test' from the terminal will take care of this for you.
-// To run the tests manually, start an instance by running docker run -p 5984:5984 couchdb:2.3.1 from a terminal.
+// To run the tests manually, start an instance by running from a terminal the following command with current directory
+// being set to "aries-framework-go":
+//     docker run -p 5984:5984 \
+//     -v $(pwd)/scripts/couchdb-config/10-single-node.ini:/opt/couchdb/etc/local.d/10-single-node.ini \
+//     -e COUCHDB_USER=admin -e COUCHDB_PASSWORD=password couchdb:3.1.0
```

Comment:
I'd rather you point devs to the `check_unit.sh` file in order for them to discover how it's done, otherwise we have to keep updating comments here and there whenever, say, the couchdb version is updated. Same issue you just ran into (having to update this comment).

Reply:
The issue I ran into earlier is due to Docker not automatically updating the couchdb image as it was updated in a previous PR..

I will update this comment to refer to `check_unit.sh`